test/parking_location/configs/parking_location_choice.csv

```diff
@@ -0,0 +1,24 @@
+Description,Expression,no_segmentation
+Person is worker,_person_is_worker@((df.person_type == 1) | (df.person_type == 3)),1
+Trip is work purpose,_work_trip@(df.dest_purpose == 'Work'),1
```

Comment:
In ActivitySim, the trip destination purpose column is just called purpose and is not capitalized.  This shows up in multiple places.